container-search/src/main/java/com/yahoo/search/query/QueryTree.java

```diff
@@ -179,4 +180,23 @@ private static void getPositiveTerms(Item item, List<IndexedItem> terms) {
         }
     }
 
+    /**
+     * @return The total number of items in this query tree
+     */
+    public int getTreeSize() {
```

Comment:
This isn't a getter so I think you should use treeSize(), or maybe countItems() to communicate that this isn't cheap.